include/interfaces/turbulence_k_omega.h

```diff
@@ -52,8 +53,47 @@ class KOmega
   void
   set_matrix_couplings(std::vector<std::string> &couplings) const;
 
+  virtual void connect_to_signals() const
+  {
+    auto &signals = this->get_signals();
+    auto &pcout = this->get_pcout();
+    signals.begin_make_grid_fe.connect(
+      [&]()
+    {
+      pcout << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl
+            << " --------------------- Eikonal Equation -----------------------"
+            << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl << std::endl;
+      distance.run ();
+      pcout << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl
+            << " --------------------- Eikonal Equation -----------------------"
+            << std::endl
+            << " --------------------------------------------------------------"
+            << std::endl << std::endl;
+      const typename LAC::VectorType &distance_sol = distance.get_solution();
+      const DoFHandler<dim,spacedim> &distance_dh  = distance.get_dof_handler();
+      Functions::FEFieldFunction<dim, DoFHandler<dim,spacedim>, typename LAC::VectorType> dist(distance_dh, distance_sol);
+      {
+        Point<spacedim> p;
+        p[0] = 0.5;
+        p[1] = 0.5;
+        pcout << "-->" << dist.value(p,0) << std::endl;
```

Comment:
An error occurred in line <3372> of file <../include/deal.II/dofs/dof_accessor.templates.h> in function
    void dealii::DoFCellAccessor<dealii::DoFHandler<2, 2>, false>::get_dof_values(const InputVector &, ForwardIterator, ForwardIterator) const [DoFHandlerType = dealii::DoFHandler<2, 2>, lda = false, InputVector = dealii::TrilinosWrappers::MPI::BlockVector, ForwardIterator = double *]
The violated condition was:
    this->is_artificial() == false
The name and call sequence of the exception was:
    ExcMessage ("Can't ask for DoF indices on artificial cells.")
Additional Information:
Can't ask for DoF indices on artificial cells.